app/addons/databases/resources.js

```diff
@@ -137,6 +137,8 @@ function(app, FauxtonAPI, Documents) {
       var updateSeq = this.get("update_seq");
       if (full || (typeof(updateSeq) === 'number')) {
         return updateSeq;
+      } else if (typeof(updateSeq) === 'string') {
+        return updateSeq.split('-')[0];
```

Comment:
I don't think it matters the type as this is just rendered. So ultimately its converted to a string. We could always convert to a string in this method to be safer.